season6/markers.js

```diff
@@ -48,7 +47,10 @@ var MAPCRAFTER_MARKERS = [
 				{"pos" : [+220, -1025, 64], "title" : "Statue of Hermitty", "icon" : "poi.png", "iconSize" : [24, 24]},
 				{"pos" : [+240, -710, 64], "title" : "Vault 6 Entrance", "icon" : "poi.png", "iconSize" : [24, 24]},
 				{"pos" : [+240, -800, 64], "title" : "Sahara Entrance", "icon" : "sahara.png", "iconSize" : [24, 24]},
-				{"pos" : [+240, -900, 64], "title" : "Stock market", "icon" : "poi.png", "iconSize" : [24, 24]},				
+				{"pos" : [+240, -900, 64], "title" : "Stock market", "icon" : "poi.png", "iconSize" : [24, 24]},
+				{"pos" : [-620, -724, 68], "title" : "Impulse's Base / iBay", "icon" : "impulse.png", "iconSize" : [24, 24]},
+				{"pos" : [-586, -684, 74], "title" : "Zedaph and Tango's Chicken Fountain", "icon" : ".png", "iconSize" : [24, 24]},
```

Comment:
missing icon. If the build by multiple players use poi.png

Reply:
> there you go! thanks!

Nice!